Tutorials/Amr/Advection_F/Exec/SingleVortex/Make.package

```diff
@@ -1,3 +1 @@
-f90EXE_sources += Prob.f90
-
-F90EXE_sources += face_velocity_$(DIM)d.F90
+f90EXE_sources += Prob_$(DIM)d.f90 face_velocity_$(DIM)d.f90
```

Comment:
This compiles for me in 3D now, but not 2D. I think the problem is a capitalization issue - it's looking for `face_velocity_2d.f90` but the file is called `face_velocity_2d.F90`. Are you testing this on a Mac? You might miss this if your file system is not case-sensitive.

Reply:
Mac - yes. Even though I had face_velocity_2d.f90 (small) on my system the git did not change it. It remained capital .F90 in the repo. Had to remove and add again. Have fixed it now. Sorry for the hassle.